pysat/instruments/templates/template_cdaweb_instrument.py

```diff
@@ -9,29 +9,37 @@
 Also a good place to provide contact information. This text will
 be included in the pysat API documentation.
 
-Parameters
+Properties
 ----------
-platform : string
+platform
     *List platform string here*
-name : string
+name
     *List name string here*
-sat_id : string
+sat_id
     *List supported sat_ids here*
-tag : string
+tag
     *List supported tag strings here*
 
 Note
 ----
-::
+- Add any notes here
 
-    Notes
 
 Warnings
 --------
+- Add any warnings here
```

Comment:
Recommend removing optional categories from template.  But there is good reason to keep them, so maybe we should discuss this.

Reply:
I was thinking of showing how this should be formatted in the templates to help move toward a uniform doc style.  One of the style errors I noticed is that if there are not two blank lines after the warnings, the next line (usually author information) gets added to the warnings box in the html.  What if there was a note that these are optional and should be removed if unused?

Reply:
Sounds good.  Also add that note about the two blank lines.